utils/aws/s3.ts

```diff
@@ -0,0 +1,29 @@
+import { s3 } from "aws/index";
+import fs from "fs";
+import path from "path";
+
+/**
+ * Uploads a file to the AWS S3 bucket corresponding to env.AWS_BUCKET_NAME
+ * @param {string} bucketName - Name of the bucket to upload the file to
+ * @param {string} filePath - Path of the file to upload
+ * @returns {promise} - S3 upload response object on resolve, error object on reject
+ */
+const uploadToS3 = (bucketName: string, filePath: string): Promise<unknown> => {
```

Comment:
This function actually isn't returning a promise anymore because you're resolving the promise within the function itself. It seems to either return `data` or `err` which or of two different types and you generally want to avoid doing that in Typescript. You have two options here: 
- Instead of resolving the promise within the function itself, you can return it from the function and let the caller resolve it. 
- Make a custom s3 response object similar to [this](https://github.com/uwblueprint/social-diversity-for-children/blob/staging/src/models/response.ts#L6-L9) and then the caller can check to see if the error field is null or not. If it's not null there's an error and if it is then there's no error and the caller can check the data field.  

The above two suggestions can be done in separate PR if you like but for now it's probably good to change the return type from `Promise<unknown>` to just `unknown`. cc @feng-eric for thoughts

Reply:
I think the second approach with using a custom s3 response object seems good. We can do it in a different PR.